projects/fal/src/fal/cli/create.py

```diff
@@ -0,0 +1,26 @@
+PROJECT_TYPES = ["app"]
+
+
+def _create_project(project_type: str):
+    from cookiecutter.main import cookiecutter
+
+    cookiecutter("https://github.com/fal-ai/cookiecutter-fal.git")
```

Comment:
The `create` name feels pretty generic, a bit worried about taking it up for from-the-template creation, but not 100% sure yet.

I think this is equivalent to `fly launch` https://fly.io/docs/flyctl/launch/ , but that's also a pretty generic name though less than `create`. WDYT? As always, naming is the hardest part :D

Reply:
the complete command is `fal create app`  at the moment. i thought this is not that much generic 😄 another alternative I had in my mind was `new`, so it would look like `fal new app`

Reply:
We could also just keep it simple and use cookie-cutter directly, it is a popular well-known tool too after all and we won't have to proxy anything that way.